Makefile

```diff
@@ -160,12 +160,10 @@ gobindata-tool: ${GOBINDATA}
 .PHONY: kops-gobindata
 kops-gobindata: gobindata-tool ${BINDATA_TARGETS}
 
-UPUP_MODELS_BINDATA_SOURCES:=$(shell find upup/models/ | egrep -v "upup/models/bindata.go")
-upup/models/bindata.go: ${GOBINDATA} ${UPUP_MODELS_BINDATA_SOURCES}
+upup/models/bindata.go: ${GOBINDATA}
```

Comment:
This target is for building `upup/models/bindata.go`. The added dependency looks to me like it is depending on itself. I don't understand why this was added.

Reply:
This task must be re-run whenever any of the model files in upup/models changes - those are files we're including.  But bindata.go is a circular dependency, and so is excluded.